### PR TITLE
initial new structure Getting started - PR to main branch

### PR DESCRIPTION
**Pull request by emirgens**

### CODE REVIEW

File: public-site/docs/src/.vuepress/config.js [hunk 1]
```diff
@@ -29,6 +29,10 @@ module.exports = {
     lastUpdated: false,
     sidebarDepth: 1,
     nav: [
+      {
```
Comment (magnus-longva-bouvet) [inline public-site/docs/src/.vuepress/config.js:32]: can't see this actually appears

File: public-site/docs/src/start/radix-concepts/index.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## Component
+
+A component represents a standalone process running within an [environment](#environment) in a Radix application. Components are defined in the [`radixconfig.yaml`](../../references/reference-radix-config/#components), but they are only instantiated by [deployments](#deployment), which specify the Docker image to use. A component can have one or more running [replicas](#replica), depending on its configuration.
+
+> Familiar with Docker or containers? A Radix component can be thought of as Docker image, and replicas as containers running that image.
+
+If a component's `publicPort` is defined, endpoints are made available on the public Internet for each environment the component is deployed to. This allows connections via HTTPS into Radix, which are routed internally to an HTTP endpoint on the component. The domain name for the public endpoint is auto-generated from the component, environment, and application names: `https://[component]-[application]-[environment].[cluster-name].radix.equinor.com`.
```
Comment (satr) [inline public-site/docs/src/start/radix-concepts/index.md:39]: "[cluster-name]" - it does not exists for Platform cluster for publicly available link when `dnsAppAlias` is used.

File: public-site/docs/src/start/radix-concepts/index.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+Radix creates a [job-scheduler](../../guides/configure-jobs#job-scheduler) service for each job defined in [`radixconfig.yaml`](../../references/reference-radix-config/#jobs). The job-scheduler is a web API that you use to create, delete and monitor containers from the Docker image built or defined for the job. HTTP requests to the job-scheduler can only be sent by components running in the same application and environment.
+
+When creating a new job, a payload with arbitrary arguments can be specified in the body of the HTTP request to the job-scheduler. The payload is a string and can therefore contain any type of data (text, json, binary) as long as you encode it as a string, e.g. base64, when sending the request to the job-scheduler, and decode it when reading it from the file in the container where the payload is mounted.
+
+Multiple job containers can run simultaneously. Each container is assigned a unique name that can be used to monitor the state of the job through the job-scheduler API. This name is also the internal DNS name that you can use to communicate with a specific job if it exposes any ports, e.g. a custom metrics HTTP endpoint.
```
Comment (satr) [inline public-site/docs/src/start/radix-concepts/index.md:53]: "Each **container** is assigned" - "Each **job** is assigned"

File: public-site/docs/src/start/radix-concepts/index.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+Radix creates a [job-scheduler](../../guides/configure-jobs#job-scheduler) service for each job defined in [`radixconfig.yaml`](../../references/reference-radix-config/#jobs). The job-scheduler is a web API that you use to create, delete and monitor containers from the Docker image built or defined for the job. HTTP requests to the job-scheduler can only be sent by components running in the same application and environment.
```
Comment (satr) [inline public-site/docs/src/start/radix-concepts/index.md:49]: "for each **job** " - "for each job-component "

File: public-site/docs/src/start/registering-app/index.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+title: Registering your application
+---
+
+We are now ready to register our application using the Radix Web Console. Load the console for the [Radix cluster](../getting-started/#the-radix-clusters) that you want to use: [Radix Playground](https://console.playground.radix.equinor.com/applications/) or [Radix Platform](https://console.radix.equinor.com/applications/). Click "Create new app" in the top right of the page and follow the instructions there to integrate the GitHub repository with Radix.
+
+Remember that we can always change the [`radixconfig.yaml`](../../references/reference-radix-config/) file and the `Dockerfile`s after registration to change how the application builds and deploys.  
+
+Here is an example of how a new application registration could look like  
+
+![CreateApplication](./create-application.png)
```
Comment (satr) [inline public-site/docs/src/start/registering-app/index.md:11]: This file does not exist

File: public-site/docs/src/start/requirements/index.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+There aren't many requirements: Radix runs applications written in Python, Java, .NET, JavaScript, or [LOLCODE](https://en.wikipedia.org/wiki/LOLCODE) equally well. If it can be built and deployed as Docker containers, we are nearly ready. If not, it's not hard to "dockerise" most applications.
+
+An in-depth understanding of Docker is not a requirement, but it helps to be familiar with the concepts of containers and images. There are many beginner tutorials online; here's one of the most straightforward: [Getting Started with Docker](https://scotch.io/tutorials/getting-started-with-docker).
+
+It is also expected that you have a [basic understanding of Git](http://rogerdudler.github.io/git-guide/) (branching, merging) and some networking (ports, domain names).
+
+### Repository
+
+A GitHub repository for our code is required (only GitHub is supported at the moment)
```
Comment (satr) [inline public-site/docs/src/start/requirements/index.md:13]: Other repositories can be used to build images, specified in Radix.

File: public-site/docs/src/start/what-is-radix/index.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+
+You can use Radix just to run code, but the main functionality is to integrate with a code repository so that it can continuously build, test, and deploy applications. For instance, Radix can react to a `git push` event, automatically start a new build, and push it to the `test` environment, ready to be tested by users.
+
+Radix also provides monitoring for applications. There are default 📈 metrics (e.g. request latency, failure rate), but you can also output custom metrics from your code. Track things that are important for your application: uploaded file size, number of results found, or user preferences. Radix collects and monitors the data.
+
+> To help improve Radix, request access to the GitHub [Omnia Radix Readers](https://github.com/orgs/equinor/teams/omnia-radix-readers/members) team — this gives you access to poke around in our repositories. We track **issues and feature requests** in the [radix-platform](https://github.com/equinor/radix-platform/issues) repo. Please log those! 🙂
```
Comment (satr) [inline public-site/docs/src/start/what-is-radix/index.md:11]: Radix is open source - no need for the access "to poke around", correct?

File: public-site/docs/src/start/workflows/index.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
+---
+title: Workflows
+---
+
+In Radix, our development workflow can be modelled to match our needs. It is possible to use, for instance [Git Flow](https://nvie.com/posts/a-successful-git-branching-model/) or [Trunk-based development](https://paulhammant.com/2013/04/05/what-is-trunk-based-development/). Radix gives us a couple of tools to shape our workflow: **branch-environment mapping** and **deployment promotion**.
+
+If we **map** a git branch to an environment, commits to that branch will trigger a build and deployment to the specified environment. For instance, a `dev` environment might be built and deployed from `master`, while a `prod` environment can be built and deployed from the `production` branch. Branches and environments that are not mapped are ignored and do not trigger automatic builds or deployments.
```
Comment (satr) [inline public-site/docs/src/start/workflows/index.md:7]: "from `master`," - "from `master` **branch**,"

File: public-site/docs/src/start/workflows/index.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+
+**Promotion** allows us to take an existing deployment in an environment, and deploy it in another. For instance, take what is currently running in the `preprod` environment and place it in `prod`.
+
+It is fine to combine these features to produce the workflow that we want. For instance, we can automatically build and deploy `master` to the `dev` environment, and the `release` branch to `preprod`. We can then manually promote the deployment in `preprod` to the `prod` environment.
```
Comment (satr) [inline public-site/docs/src/start/workflows/index.md:11]: "deploy `master` to" - "deploy `master` **branch** to"

File: public-site/docs/src/start/workflows/index.md [hunk 10]
```diff
@@ -0,0 +1,14 @@
+---
+title: Workflows
+---
+
+In Radix, our development workflow can be modelled to match our needs. It is possible to use, for instance [Git Flow](https://nvie.com/posts/a-successful-git-branching-model/) or [Trunk-based development](https://paulhammant.com/2013/04/05/what-is-trunk-based-development/). Radix gives us a couple of tools to shape our workflow: **branch-environment mapping** and **deployment promotion**.
+
+If we **map** a git branch to an environment, commits to that branch will trigger a build and deployment to the specified environment. For instance, a `dev` environment might be built and deployed from `master`, while a `prod` environment can be built and deployed from the `production` branch. Branches and environments that are not mapped are ignored and do not trigger automatic builds or deployments.
+
+**Promotion** allows us to take an existing deployment in an environment, and deploy it in another. For instance, take what is currently running in the `preprod` environment and place it in `prod`.
```
Comment (satr) [inline public-site/docs/src/start/workflows/index.md:9]: It is also possible to promote previous non-active deployment of the same environment - "rollback" feature.

**Review by satr — CHANGES_REQUESTED** [review]

**Pull request merged**